src/server/webkit/wkPage.ts

```diff
@@ -966,8 +966,11 @@ export class WKPage implements PageDelegate {
     // Under certain conditions we never get the Network.responseReceived
     // event from protocol. @see https://crbug.com/883475
     const response = request.request._existingResponse();
-    if (response)
+    if (response) {
```

Comment:
Do we have `protocol` available in WebKit?

Reply:
Good catch, we need to add that at `src/server/webkit/wkInterceptableRequest.ts:113`

Reply:
The issue is it only becomes available later in the cycle. The `metrics` property (which holds the `protocol` gets populated in the `loadingFinishedPayload` at `src/server/webkit/protocol.ts:5475`

Reply:
@dgozman this was actually not that hard to fix. I added a patch for it in the last commit. We are now using the http protocol reported by webkit.